crates/analyzer/src/analyzer.rs

```diff
@@ -395,7 +439,8 @@ fn traverse_assignable_symbol(id: SymbolId, path: &AssignPath) -> Vec<AssignPath
             }
             SymbolKind::Variable(x)
                 if x.affiniation == VariableAffiniation::Module
-                    || x.affiniation == VariableAffiniation::Function =>
```

Comment:
Sorry. I misspelled "affiliation" as "affiniation" in `symbol::VariableAffiniation`. "affiliation" is correct.